src/Pods/Data/Map_Field_Values.php

```diff
@@ -703,6 +704,85 @@ public function image_fields( $field, $traverse, $field_data, $obj ) {
 		return pods_traverse( $traverse_params, $value );
 	}
 
+	/**
+	 * Map the date field value.
+	 *
+	 * @since 2.9.14
+	 *
+	 * @param string                  $field      The first field name in the path.
+	 * @param string[]                $traverse   The list of fields in the path excluding the first field name.
+	 * @param null|Field|Object_Field $field_data The field data or null if not a field.
+	 * @param Pods|null               $obj        The Pods object or null if not set.
+	 *
+	 * @return null|mixed The date field value or null if there was no match.
+	 */
+	public function date_fields( $field, $traverse, $field_data, $obj = null ) {
+
+		// Default date field handlers.
+		$date_fields = array(
```

Comment:
FYI we could use `PodsForm::date_field_types()` here but because it makes more sense to only keep this for date/time/datetime as kind of global special tags, I don't think it needs to match the current date field types at all and can remain as this hardcoded list.

This is more of a note for ourselves to reference for why this is here :)